customizations/README.md

```diff
@@ -379,6 +384,201 @@ externalRedis:
     #   ...
     #   -----END CERTIFICATE-----
 ```
+### Custom Buckets
+
+For every CARTO Self Hosted installation, we create GCS buckets in our side as part of the required infrastructure for importing data, map thumbnails and other internal data.
+
+You can create and use your own storage buckets in any of the following supported storage providers:
+
+- Google Cloud Storage
+- AWS S3
+- Azure Storage
+
+> :warning: You can only set one provider at a time.
+
+<!--
+TODO: Add the code related to Terraform
```

Comment:
Instead of the code, just add a link to the proper `.tf` file in `carto-selfhosted/terraform/examples`